server/configmodule.py

```diff
@@ -96,6 +99,7 @@ class MinikubeConfig(Config):
 
 class LocalConfig(Config):
     LOCAL = True
+    ADMIN = True
```

Comment:
the local private dc's should have this set back to False since they won't have access to secret manager.

Reply:
Updated this with refactored base class for local and private instance now.